v20.2/set-transaction.md

```diff
@@ -22,7 +22,8 @@ Parameter | Description
 ----------|------------
 `PRIORITY` | If you do not want the transaction to run with `NORMAL` priority, you can set it to `LOW` or `HIGH`.<br><br>Transactions with higher priority are less likely to need to be retried.<br><br>For more information, see [Transactions: Priorities](transactions.html#transaction-priorities).<br><br>The current priority is also exposed as the [session variable](show-vars.html) `transaction_priority`.<br><br>**Default**: `NORMAL`
 `READ` | Set the transaction access mode to `READ ONLY` or `READ WRITE`. The current transaction access mode is also exposed as the [session variable](show-vars.html) `transaction_read_only`.<br><br>**Default**: `READ WRITE`
-`AS OF SYSTEM TIME` | Execute the transaction using the database contents "as of" a specified time in the past.<br/><br/> The `AS OF SYSTEM TIME` clause can be used only when the transaction is read-only. If the transaction contains any writes, or if the `READ WRITE` mode is specified, an error will be returned.<br/><br/>For more information, see [AS OF SYSTEM TIME](as-of-system-time.html).<br/><br/>
+`AS OF SYSTEM TIME` | Execute the transaction using the database contents "as of" a specified time in the past.<br/><br/> The `AS OF SYSTEM TIME` clause can be used only when the transaction is read-only. If the transaction contains any writes, or if the `READ WRITE` mode is specified, an error will be returned.<br/><br/>For more information, see [AS OF SYSTEM TIME](as-of-system-time.html).
+`NOT DEFERRABLE`<br>`DEFERRABLE` | <span class="version-tag">New in v20.2:</span> This clause is a no-op, supported for compatibility with PostgreSQL. `NOT DEFERRABLE` is the default behavior. `DEFERRABLE` returns an `unimplemented` error.
```

Comment:
returning an error isn't classified as a no-op i think. `NOT DEFERABLE` no-ops and is the default behaviour.`

otherwise LGTM

Reply:
Got it. Updated again. Thanks for iterating on this!